### PR TITLE
API-1437 java 12 upgrade

### PR DESCRIPTION
**Pull request by jblefkove-va**
https://vasdvp.atlassian.net/browse/API-1437

Merge after https://github.com/department-of-veterans-affairs/health-apis-eligibility-and-enrollment-client/pull/22

### CODE REVIEW

File: hand-of-queen-elizabeth/src/main/java/gov/va/api/health/queenelizabeth/ee/impl/ValidateEligibilities.java [hunk 1]
```diff
@@ -26,31 +26,21 @@ private Document parse(SoapMessageGenerator soapMessageGenerator, String xml) {
...
+    Document document = parse(messageGenerator, xml);
     XmlResponseValidator.builder()
-        .soapMessageGenerator(originalSoapMessage)
-        .response(xml)
+        .soapMessageGenerator(messageGenerator)
+        .response(document)
         .build()
         .validate();
-    return write(originalSoapMessage, xml);
+    return xml;
```
Comment (jblefkove-va) [inline hand-of-queen-elizabeth/src/main/java/gov/va/api/health/queenelizabeth/ee/impl/ValidateEligibilities.java]: While testing community-care-eligibility on Java 12, I noticed that responses from E&E were no longer marshaling. I observed that in this method, due to (I suspect) some library/dependency change, the namespace was being dropped from the response in this method. Specifically, the response string from the original SOAP call looks like this:
```
<?xml version="1.0" encoding="UTF-8"?>
<getEESummaryResponse xmlns="http://jaxws.webservices.esr.med.va.gov/schemas">
...
</getEESummaryResponse>
```

But after unmarshaling it into a document, validating, and marshaling back, the returned value from this method is:
```
<?xml version="1.0" encoding="UTF-8"?>
<getEESummaryResponse>
...
</getEESummaryResponse>
 ```

The missing namespace means that CCE can no longer unmarshal the response once it gets over there.

The solution was to have this method return the original result of the SOAP call. (Validation still happens, but does not alter the response.)

**Comment by bschofield-va** [comment]
What's going on with https://github.com/department-of-veterans-affairs/health-apis-eligibility-and-enrollment-client/pull/22

**Comment by dtournour-va** [comment]
I just realized both teams tackled this.  hmmm.

**Comment by alecbert-va** [comment]
Sorry that was my bad. I hadn't talked to Peter about it because I thought QE was our responsibility.

**Comment by jblefkove-va** [comment]
Bump

**Pull request merged**